Info.lua

```diff
@@ -697,6 +715,24 @@ g_PluginInfo =
 			},
 		},
 
+		
+		["gamerule"]=
+		{
+			Handler = HandleConsoleGameRule,
+			HelpString = "Sets or queries a game rule value.",
+			ParameterCombinations =
+			{
+				{
+					Params = "rule",
+					Help = "Queries a game rule value",
+				},
+				{
+					Params = "rule value",
+					Help = "Sets a game rule value",
```

Comment:
The params should be correct from the start, even if not used. Makes migration easier.